libraries/chain/controller.cpp

```diff
@@ -750,6 +750,24 @@ struct controller_impl {
          // else no checks needed since fork_db will be completely reset on replay anyway
       }
 
+      if (auto dm_logger = get_deep_mind_logger()) {
+         // FIXME: We should probably feed that from CMake directly somehow ...
+         fc_dlog(*dm_logger, "DEEP_MIND_VERSION 12");
+
+         fc_dlog(*dm_logger, "ABIDUMP START");
```

Comment:
Note also that we decided against using a flag for the initial ABIs dump. Mainly because the operation is really fast and the actual ABI serialization cannot work properly without the ABIs dump, so we see no reason to make it behind a flag.